### ISSUE TITLE
Suggestion&Request: Updated / tracker for py -V

### ISSUE THREAD

**Issue by leomaxwell973** [ali-vilab/VGen#135]
So, I'm convinced the only way to get this working on Windows as some have, is to do some very specific by the book and specific off the book things. One, i'll guess by the book is the python version. Given the critical nature of python for some environments to work in some projects, if not all systems for a few of those even, whether you're linux or windows, etc : It would be nice to have a clear status on it. I do see in the tutorial that it's suggestion 3.8 but i'm weary about 3.9 even on this particular build since its sensitive if you're on windows it seems, least more so, that and ppl know without needing to hoop through troubleshooting for themselves, and could probably pass that snippet to any future projects i'd think so...

long story short in my thoughts:

What ver python we rockin on here these days? still 3.8? or is 3.9 in the cards? :P 

(poked about the readme, saw a reference to v.38 but no reason to think it's up to date, apologies if it is)
